nemoguardrails/library/domain_hallucination/actions.py

```
@@ -0,0 +1,92 @@
+# SPDX-License-Identifier: Apache-2.0
+
+"""NeMo Guardrails actions for domain hallucination detection (Layer 1 priority)."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any, Dict, Optional
+
+try:
+    from nemoguardrails.actions import action
+except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)